classroom-dashboard/app/views/sections/_form.html.erb

```diff
@@ -2,7 +2,7 @@
 <br>
 <br>
 <br>
-<!-- fix this to better form lol  -->
+<!-- fix this to better form lol <3 -->
```

Comment:
As comment indicates, we need to figure out how to space this form on the page without manual <br>. 

We need to figure out what exact fields Dodds wants for the new Section, but since he's hard to reach, for now we probably should decide as a group what we want (year? semester?) and go with it, deleting extra parameters or adding more as needed.